docs/en/operations/table_engines/graphitemergetree.md

```diff
@@ -75,6 +75,23 @@ Rollup configuration structure:
 
 ```
 required-columns
+patterns
+```
+
+### Required columns
+
+Set the following parameters only if the name of columns differs from their default values:
```

Comment:
Needs some extra introduction: these columns have to be present in a table to make it work and it's imporant, while the settings to override their names are kind of secondary and could be introduced later.

Reply:
This introduction is in `Creating a Table` section.